### ISSUE TITLE
Create CHROOT directory before copy COMP_FILENAME

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1119]
Originally reported of Google Code with ID 1063.

```
I have tested devel/build_chroot script file on Ubuntu Precise after Hrvoje Ribicic <riba@google.com> updates it (http://git.ganeti.org/?p=ganeti.git;a=commit;h=40f22918d3e454bae0887a859a4b754bcb45c90f).

There is still a problem after I run the script. I cannot run command "sudo cp /home/gangbiao/workshop2/ganeti/devel/wheezy-amd64.tar.gz /srv/chroot/wheezy-amd64.tar.gz" before I create "/srv/chroot" directory. However, the script only tells me as follows.

{{{
[...]
Setting up debhelper (9.20120909) ...
Setting up diffstat (1.55-3) ...
Setting up quilt (0.60-2) ...
update-alternatives: using /usr/bin/vim.basic to provide /usr/bin/editor (editor) in manual mode
Creating compressed schroot image...
Chroot created. In order to run it:
 * sudo cp wheezy-amd64.conf /etc/schroot/chroot.d/wheezy-amd64.conf
 * sudo cp /home/harry/workshop2/ganeti/devel/wheezy-amd64.tar.gz /srv/chroot/wheezy-amd64.tar.gz
Then run "schroot -c wheezy-amd64"
}}}

I think it need to tell users to create "/srv/chroot" before "sudo cp /home/harry/workshop2/ganeti/devel/wheezy-amd64.tar.gz /srv/chroot/wheezy-amd64.tar.gz".
```
Originally added on 2015-04-13 12:01:22 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in:

commit 56854ba6c4d412f37aa5e541f81796c0e9e0dca7
Author: Gangbiao Liu <lgb.nwpu@gmail.com>
Date:   Wed Apr 15 11:56:55 2015 +0800

    Create CHROOT directory before copy COMP_FILENAME
    
    In devel/build_chroot script file, one cannot copy $COMP_FILEPATH to $CHROOT_DIR/$COMP_FILENAME
    before create $CHROOT_DIR.
    
    Signed-off-by: Gangbiao Liu <lgb.nwpu@gmail.com>
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2015-04-15 13:55:25 +0000 UTC.

Added Labels: SmallTask Priority-Low
Changed State: Fixed
Added to Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:03 +0000 UTC.

Changed State: Released
